coreAPI/src/test/java/PomPropertiesTest.java

```diff
@@ -0,0 +1,20 @@
+import net.java.games.input.PomProperties;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class PomPropertiesTest {
```

Comment:
These tests could be more thorough, but we would need to do some ugly mocking of the `getResourceAsStream` function to pass-in an `InputStream` with test data. I can implement this if we want, but I'm open to other suggestions if there's a more straight forward way to pass in test data to the `PomProperties.loadProperties()` function.